README.md

```diff
@@ -70,6 +68,8 @@ In `Java` files, you will benefit with:
 
 The following settings are supported:
 
+* See all [microprofile.tools.* settings](https://github.com/redhat-developer/vscode-microprofile#supported-vs-code-settings) for `MicroProfile/Quarkus` support in `properties` and `Java` files.
```

Comment:
Can we change this too:

* All [microprofile.tools.* settings](https://github.com/redhat-developer/vscode-microprofile#supported-vs-code-settings) from the [Visual Studio Code extension for MicroProfile](https://github.com/redhat-developer/vscode-microprofile)

Reply:
fixed